docs/src/plotting_functions/barplot.md

```diff
@@ -22,3 +22,73 @@ barplot!(xs, ys .- 1, fillto = -1, color = xs, strokecolor = :black, strokewidth
 
 f
 ```
+
+```@example bar
+using CairoMakie
+CairoMakie.activate!() # hide
+AbstractPlotting.inline!(true) # hide
+
+tbl = (x = [1, 1, 1, 2, 2, 2, 3, 3, 3],
+	   height = 0.1:0.1:0.9,
+	   grp = [1, 2, 3, 1, 2, 3, 1, 2, 3],
+	   grp1 = [1, 2, 2, 1, 1, 2, 1, 1, 2],
+	   grp2 = [1, 1, 2, 1, 2, 1, 1, 2, 1]
+	)
+```
+
+```@example bar
+barplot(tbl.x, tbl.height,
+		stack = tbl.grp,
+		color = tbl.grp,
+		axis = (xticks = (1:3, ["left", "middle", "right"]),
+				title = "Stacked bars"),
+        figure = (resolution = (800, 600), )
+		)
```

Comment:
It may be worth it to fix the indentation (here and below), as these code snippets will show up in the docs.

Reply:
it's really odd. In vs code everything is nicely aligned. 
![image](https://user-images.githubusercontent.com/6280307/114862690-1f220580-9def-11eb-8f20-b36d54d8a165.png)

Reply:
fixed